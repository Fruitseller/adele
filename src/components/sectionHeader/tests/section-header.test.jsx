import React from 'react';
import 'jest-styled-components';

import SectionHeader from '../section-header';

it('renders SectionHeader correctly', () => {
  const wrapper = mount(<SectionHeader content="why adele?" />);
  expect(wrapper).toMatchSnapshot();
});
