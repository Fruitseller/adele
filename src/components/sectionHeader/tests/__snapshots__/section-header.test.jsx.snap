// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SectionHeader correctly 1`] = `
.c0 {
  font-family: Proxima Nova;
  font-size: 14px;
  font-weight: 100;
  color: #777777;
  font-variant: small-caps;
  margin: 80px 0 40px 0;
}

<SectionHeader
  content="why adele?"
>
  <styled.h3>
    <h3
      className="c0"
    >
      why adele?
    </h3>
  </styled.h3>
</SectionHeader>
`;
